### PR TITLE
features: added connex static freqencies, you can set rssithreshold directly, you can set any frequency you want directly

### PR DESCRIPTION
**Pull request by anunique**
added connex static frequencies

### CODE REVIEW

File: Arduino/chorus-rf-laptimer/chorus-rf-laptimer.ino [hunk 1]
```diff
@@ -770,7 +770,7 @@ void setChannel(uint8_t channel) {
 }
 // ----------------------------------------------------------------------------
 void incBand() {
-    if (bandIndex < 5) {
+    if (bandIndex < 7) {
```
Comment (voroshkov) [inline Arduino/chorus-rf-laptimer/chorus-rf-laptimer.ino]: also need to update the "setBand" function

Comment (voroshkov) [inline Arduino/chorus-rf-laptimer/chorus-rf-laptimer.ino]: or better refactor and move the MAX_BAND to const

**Comment by anunique** [comment]
sounds like a plan ;)

**Comment by anunique** [comment]
bunch of new features here:

- connex frequencies are added to the static channel/band table
- it allows setting the frequency directly, old way of setting channel and band still is in there for compatiblity reasons
- you can set the threshold, old way still is in there for compatiblity reasons

File: Arduino/chorus-rf-laptimer/pinAssignments.h [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
+*/
+
+// Pin definitions
+
+#define led 13
+#define spiDataPin 10
+#define slaveSelectPin 11
+#define spiClockPin 12
+#define pinRaspiInt 5 //digital
+#define rssiPinA 1  //analog
```
Comment (voroshkov) [inline Arduino/chorus-rf-laptimer/pinAssignments.h]: why do you change the rssi pin? it will break existing schematic and PCBs

Comment (anunique) [inline Arduino/chorus-rf-laptimer/pinAssignments.h]: just ignore it. i plugged those receivers onto the arduino directly hehehe

**Pull request merged**